docs/howto/managedidentity.md

```diff
@@ -57,33 +65,41 @@ spec:
 EOF
 ```
 
-4. Install [aad-pod-identity](https://github.com/Azure/aad-pod-identity#1-create-the-deployment)
+Let's break down exactly what the script is doing.
 
-```shell
-make install-aad-pod-identity
-```
+1. Install [aad-pod-identity](https://github.com/Azure/aad-pod-identity#1-create-the-deployment).
```

Comment:
there's no anchor for `#1-create-the-deployment`, perhaps we want `#1-deploy-aad-pod-identity`?